### PR TITLE
change: check for "master_on" for test cases is no needed in ngx-releng.

### PR DESCRIPTION
**Pull request by zhuizhuhaomeng**

### CODE REVIEW

File: ngx-releng [hunk 1]
```diff
@@ -237,7 +237,7 @@ if [ -d src ]; then
     tfiles=`find -L t -name '*.t'`
 
     ack -l '\r\n' $tfiles /dev/null
-    ack '^master_on' $tfiles /dev/null
+    #ack '^master_on' $tfiles /dev/null
```
Comment (spacewander) [inline ngx-releng:240]: Why not remove it?

Comment (zhuizhuhaomeng) [reply ngx-releng:240]: https://github.com/openresty/lua-nginx-module/pull/1916

Comment (zhuizhuhaomeng) [reply ngx-releng:240]: I would like to add the auto check for the ci.
some cases should have master_on， checking for it is meaningless.

Comment (freecodelover) [reply ngx-releng:240]: I also think remove it simply could be better.

Comment (zhuizhuhaomeng) [reply ngx-releng]: has removed it

**Review by xiaocang — APPROVED** [review]

**Pull request merged**